client/src/pubsub_client.rs

```diff
@@ -98,7 +98,7 @@ where
     }
 
     pub fn send_unsubscribe(&self) -> Result<(), PubsubClientError> {
-        let method = format!("{}Unubscribe", self.operation);
+        let method = format!("{}Unsubscribe", self.operation);
```

Comment:
:eyes:  @CriesofCarrots

Reply:
Yowza, been like that since the beginning! 😝